.github/workflows/build.yml

```diff
@@ -36,6 +36,7 @@ jobs:
                   selinux-policy-devel \
                   systemd-devel \
                   valgrind \
+                  git-all \
```

Comment:
Isn't git installed by default on GH? Otherwise there would not be a way to checkout sources ...

Reply:
It's only attempted as the fallback if `.clang-format-include` etc. do not exist.

Reply:
Thanks! I created a PR to add the include file and remove the git dependency (#240)